tests/storage-compat/gen_storage_compat_data.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+# This test checks that Qdrant answers to all API mentioned in README.md as expected
+
+set -ex
+
+QDRANT_HOST='localhost:6333'
+
+# cleanup collection if it exists
+curl -X DELETE "http://$QDRANT_HOST/collections/test_collection" \
+  -H 'Content-Type: application/json' \
+  --fail -s | jq
+
+# create collection
+curl -X PUT "http://$QDRANT_HOST/collections/test_collection" \
+  -H 'Content-Type: application/json' \
+  --fail -s \
+  --data-raw '{
+      "vector_size": 50,
+      "distance": "Dot",
+      "optimizers_config": {
+        "default_segment_number": 2
```

Comment:
```suggestion
        "default_segment_number": 2,
        "indexing_threshold_kb": 10
```

Reply:
Just understood this comment :+1: 
I will apply this change and update the docs to remove the manual config change step.

Reply:
https://github.com/qdrant/qdrant/pull/875/commits/ccbedfe71a736772242f04fc03be81327156726b